src/App.js

```diff
@@ -1,11 +1,16 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is quality and clean.

Reply:
@rloterh Sir, I have enabled and fixed the errors.